Carbon.MassTransit/Carbon.MassTransit.csproj

```diff
@@ -2,8 +2,10 @@
 
 	<PropertyGroup>
 		<TargetFramework>netstandard2.0</TargetFramework>
-		<Version>3.1.1</Version>
-		<Description>3.1.1
+		<Version>3.3.0</Version>
```

Comment:
Well, it is used as a beta version for routing slip in my local :) Not to have any conflicts, skipped to 3.3.0, rather than making it 3.2.10 or sth else.